mypy.ini

```diff
@@ -0,0 +1,40 @@
+[mypy]
+disallow_untyped_calls = True
+disallow_untyped_defs = True
+disallow_incomplete_defs = True
+check_untyped_defs = True
+disallow_subclassing_any = True
+warn_no_return = True
+strict_optional = True
+strict_equality = True
+no_implicit_optional = True
+disallow_any_generics = True
+disallow_any_unimported = False
+warn_redundant_casts = True
+warn_unused_ignores = True
+warn_unused_configs = True
+show_traceback = True
+show_error_codes = True
+pretty = True
+python_version = 3.7
```

Comment:
Above are the global settings. Then we have a INI group setting style where we can have `[mypy-${MODULE-NAME}]` to apply settings per modules.

Below we ignore the imported modules that apparently do not have types yet.